server/api/support.py

```diff
@@ -419,3 +419,24 @@ def log_in_as_jurisdiction_admin(email: str):
         session, UserType.JURISDICTION_ADMIN, email, from_support_user=True
     )
     return redirect("/")
+
+
+@api.route("/support/elections/<election_id>/rounds/current/reopen", methods=["PATCH"])
```

Comment:
For the support API, I've been trying to keep the paths as shallow as possible (instead of having nested resource paths with multiple IDs in them). It's an experiment to see if that ends up being cleaner than the long paths in the main Arlo APIs.

So to follow the pattern, I'd probably make this one:
```
PATCH /support/rounds/<round_id>/reopen
```
or
```
DELETE /support/rounds/<round_id>/ended-at
```
What do you think?

Reply:
Ah interesting! I chose to use election ID in the path here because we only ever want to reopen the current round. Rather than having the client identify and provide the current round, I figured it'd be easier to have the server look up the current round ID itself. On that note, I intentionally didn't include round ID in the path

Reply:
That makes sense to me. What do you think about shortening the path then to something like `/elections/<election_id>/reopen-round`? I think when I saw the multiple segments `/rounds/current/reopen` it suggested to me that we would be building a nested structure where `current` could be replaced by a round id for other round-based operations.

Reply:
Or maybe `/elections/<election_id>/reopen-current-round`

Reply:
Yes shortening the path sounds good to me!